### PR TITLE
Pless symmetry, quasi-twisted, parity check, quadratic residue (stub), duadic (stub)

### PR DESCRIPTION
**Pull request by cbclayton**

### CODE REVIEW

**Comment by cbclayton** [comment]
> @cbclayton FYI this isn't yet a pull request as it is not appearing on https://github.com/errorcorrectionzoo/eczoo_data/pulls

When I go to that link I see this pull request listed with "#167 opened on Mar 30 by [cbclayton]." I'm not sure what I need to do

**Comment by valbert4** [comment]
all good; i see it now

File: codes/quantum/qubits/cat-like_2n_2n-2_2.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  \end{align}
+  for all strings \(x\in\{0,1\}^{2n}\) of weight \(n\). There are \(2^{2n-2}\) such states, hence \(k=2n-2\).
+
+protection: 'Detects a single-qubit error or single erasure. Not able to correct arbitrary single-qubit errors because \( \lfloor \frac{d-1}{2} \rfloor =0 \).'
+
+
+relations:
+  parents:
+    - code_id: qubit_stabilizer
+  cousins:
```
Comment (valbert4) [inline codes/quantum/qubits/cat-like_2n_2n-2_2.yml:35]: add cousin code_id: group_gkp detail: 'Cat-like codes can be extended to group-valued qudits \cite{arXiv:1902.07714}.'

File: codes/quantum/qubits/cat-like_2n_2n-2_2.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - code_id: cat
```
Comment (valbert4) [inline codes/quantum/qubits/cat-like_2n_2n-2_2.yml:36]: Can't tell if this makes it more or less confusing. Maybe mention that cat code states are bosonic cat states while here the Schrodinger cat states, with the difference being that the latter are entangled.

File: codes/quantum/qubits/gen_divisible.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  The code has parameters \([[O(d^{\nu}),\Omega(d),d]]\).
+  This forms a tower of generalized divisible quantum codes.
+
+  Although it is valid to think of this procedure\cite{arXiv:1709.08658} as mapping binary matrices with a 
+  certain divisibilty to larger binary matrices with a higher divisibilty,
+  the construction is based on converting two magic state distillation protocols 
+  based on Clifford measurements to those based on codes with transversal T-gates.
+
+features:
+  transversal_gates: |
```
Comment (valbert4) [inline codes/quantum/qubits/gen_divisible.yml:28]: If these are somehow related to magic state distillation, can try to add some info in the magic_scaling_exponent feature. Check the list to see related codes: https://errorcorrectionzoo.org/list/quantum_magic

File: codes/quantum/qubits/gen_divisible.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  parents:
+    - code_id: qubit_stabilizer
+    - code_id: css
+  cousins:
+    - code_id: quantum_reed_muller
+      detail: 'The tower constructed by starting with the Steane [\([7,1,3]]\) code gives the family of quantum Reed-Muller codes of distance 3'
+    - code_id: steane
+      detail: 'The tower constructed by starting with the Steane [\([7,1,3]]\) code gives the family of quantum Reed-Muller codes of distance 3'
+  #  - code_id: classical divisible 
+  #child: Triply even CSS codes occur as a subclass of the tower of generalized divisible quantum codes.
```
Comment (valbert4) [inline codes/quantum/qubits/gen_divisible.yml:42]: css is only parent. uncomment the above and change code_id to divisible.

File: codes/quantum/qubits/gen_divisible.yml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+#######################################################
```
Comment (valbert4) [inline codes/quantum/qubits/gen_divisible.yml:1]: There was just a new entry submitted for related codes:
https://errorcorrectionzoo.org/c/quantum_divisible
Any simple relation to these codes? They both seem to deal with divisibility, but that's already explained by cousin connection to divisible. If you can't see anything simple, nbd.

File: codes/quantum/qubits/gen_divisible.yml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: gen_divisible
+physical: qubits
+logical: qubits
+
+name: 'Generalized divisible codes'
```
Comment (valbert4) [inline codes/quantum/qubits/gen_divisible.yml:10]: Generalized quantum divisible code

File: codes/quantum/qubits/small/stab_4_2_2.yml [hunk 7]
```diff
@@ -48,6 +48,8 @@ relations:
       detail: '\([[4,1,2]]\) subcode \(\{|\overline{00}\rangle,|\overline{01}\rangle\}\) is the smallest member of the sub-family of \([[m^2,1,m]]\) QPC codes.'
     - code_id: surface
       detail: '\([[4,2,2]]\) code is the smallest toric code.'
+    - code_id: cat-like_2n_2n-2_2
```
Comment (valbert4) [inline codes/quantum/qubits/small/stab_4_2_2.yml:51]: Nice

File: codes/quantum/qubits/gen_divisible.yml (hunk 4 above)
Comment (valbert4) [reply codes/quantum/qubits/gen_divisible.yml:42]: I've added this as parent to triply even listitng in spreadsheet; can remove

**Comment by valbert4** [comment]
Thanks, please resubmit with quantum comments addressed.

**Pull request closed without merging**